core/include/jml/math/vector.hpp

```diff
@@ -18,26 +18,25 @@ class JML_API Vector {
 
     int length; // The number of entries in the vector
 
-    double * entries;
-
-public:
+    double *entries;
 
-     Vector(int length);
-     Vector(const Vector& other);
+  public:
+    Vector(int length);
+    Vector(const Vector &other);
     ~Vector();
 
     // This applies a given function to every component in the vector.
-    void apply(ActivationFunction * fn);
+    void apply(ActivationFunction *fn);
 
     // This adds every entry in v to this vector.
-    void add(Vector& v);
+    void add(Vector &v);
 
     int get_size() const;
 
     double get_entry(int pos) const;
     void set_entry(int pos, double val);
-    void add_entry(int pos, double val); // Adds val to the current value in pos.
-
+    void add_entry(int pos,
+                   double val); // Adds val to the current value in pos.
```

Comment:
Any idea where this line break came from?

Reply:
I thin from the comment. Maybe the comment should be above the line?

Reply:
Yeah, probably. I think this is fine for now, unless you think otherwise